### PR TITLE
Add a github token to the credential store

### LINKED ISSUE #10: Provide a way to set GITHUB_TOKEN in the jobs
As, the job require to communicate with Github APIs, unauthenticated api requests are [60 per hour](https://developer.github.com/v3/#rate-limiting).

The solution is to add `GITHUB_TOKEN` in your [system for command line use](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).

Also, I couldn't find any documentation on it in JJB.

@bstinsonmhk,  I think it need admin's interaction to create token as environment variables on jenkins.

### PR DESCRIPTION
**Pull request by bstinsonmhk**
Fixes #10

### CODE REVIEW

File: minishift-ci-index.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@
         - ansicolor
         - workspace-cleanup
         - timestamps
+        - credentials-binding:
+            - text:
+                credential-id: 1a087ac4-4a16-4d26-99f5-197a9ec00c6e
```
Comment (budhrg) [inline minishift-ci-index.yaml]: Hi @bstinsonmhk , I think checking in this type of access token in code is not recommended as these acts as password. For more please refer https://help.github.com/articles/creating-an-access-token-for-command-line-use/

Comment (bstinsonmhk) [inline minishift-ci-index.yaml]: This isn't the access token, this is an index to the Jenkins credential store where the token is actually located.

Comment (budhrg) [inline minishift-ci-index.yaml]: @bstinsonmhk Ah, didn't know this kind of mechanism :smile: Thanks for the information.

**Comment by budhrg** [comment]
Merged and thanks

**Pull request merged**